TodoListPosta/app/containers/App.js

```diff
@@ -0,0 +1,87 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux'
+import {Platform, StyleSheet, Text, View} from 'react-native';
+
+import Title from '../components/title/index';
+import Footer from '../components/footer/index';
+import List from '../components/list/index';
+import Input from '../components/input/index';
+import actionCreators from '../redux/actions';
+
+// const items = [{
+//   label: 'Check email',
+//   completed: true
+//   } , {
+//   label: 'Take a shower',
+//   completed: false
+//   } , {
+//   label: 'Brush teeth',
+//   completed: false
+//   } , {
+//   label: 'Eat breakfast',
+//   completed: false
+//   } , {
+//   label: 'Take a shower',
+//   completed: true
+//   } , {
+//   label: 'Brush teeth',
+//   completed: false
+//   } , {
+//   label: 'Eat breakfast',
+//   completed: true
+// }];
+
+class TodoApp extends Component {
+  removeCompleted = () => {
+    this.props.dispatch(actionCreators.removeCompleted());
+  }
+
+  toggleItemCompleted = (index) => {
+    this.props.dispatch(actionCreators.toggleItemCompleted(index));
+  }
+
+  onRemoveItem = (index) => {
+    this.props.dispatch(actionCreators.removeItem(index));
+  }
+
+  addItem = (item) => {
+    this.props.dispatch(actionCreators.addItem(item));
+  }
+  
+  render() {
+    return (
+      <View style={styles.container}>
+        <Title> Todo List </Title>
```

Comment:
Usually, we extract these literals to a `strings.js` or a `i18n.js` file within this folder